prboom2/src/SDL/i_sound.c

```diff
@@ -1282,6 +1283,7 @@ static const music_player_t *music_players[] =
   &db_player, // dumbplayer.h
   &fl_player, // flplayer.h
   &opl_synth_player, // oplplayer.h
+  &alsa_player, // alsaplayer.h
```

Comment:
Is the order important here?

Reply:
It isn't. I don't know how I missed that instance in particular, but it still seemed to work the way it was supposed to in the UI (Portmidi before ALSA).